src/utils/authcore.ts

```
@@ -51,17 +57,9 @@ export async function initAuthcore(
       };
     });
   } else {
```

Should guard/throw error on `accessToken` and `containerId` both undefined?